### ISSUE TITLE
Open Collective Integration

### ISSUE THREAD

**Issue by jessykate** [cobudget/cobudget#93]
Explore integrating with [Open Collective](https://github.com/opencollective) such that a user/group with an Open Collective could convert their Open Collective balance into a cobudget group, make decisions about how to spend the money in Cobudget, and then have the payments and invoicing handled through Open Collective again. 

How does this work if the Open Collective balance is changing (eg. continuing to receive contributions)?

**Comment by susanbasterfield** [comment]
yes I'd like to see this for the open.collective reinventingstartups project.  We are using the platform for our pay-what-you-feel download of the book.  The idea is that once the funds hit a certain level, we'd like to offer a cobudget process to the community of supporters/contributors.  It would be rad if that was a cobudget plug-in or even integration.

**Comment by chime-mu** [comment]
Inspired by Frans adventures in Platform Design, I just read this: https://meedabyte.com/2015/11/06/platform-design-toolkit-2-0-open-for-comments/#Important - which makes a lot of sense.
So we should see this ability to interface with other innovative finance tools as a strategic capability. Or at least that's what I've persuaded myself to believe right now ...
